app/presenters/nav_tab/link_collection.rb

```diff
@@ -38,10 +38,21 @@ def default
 
   private
 
+  def payment_sources
+    NavTab::Link.new(
+        tab: :payment_sources,
+        subnav: [:accounts, :transactions].map { |method_name| send(method_name) },
```

Comment:
I think this would be clearer: `subnav: [accounts, transactions]`. I don't think we need the metaprogramming.

I might even be ok with going fully inline:

```ruby
tab: :payment_sources,
subnav:
  [
    NavTab::Link.new(tab: :accounts, text: t_my(Account), url: accounts_path),
    NavTab::Link.new(tab: :transactions, text: I18n.t("pages.transactions"), url: transactions_path),
  ]
```